fixchain/fixer.go

```diff
@@ -107,7 +107,7 @@ func min(a, b int) int {
 	return b
 }
 
-// sort.Sort(data Interface) for chainSlice - uses data.Len, data.Less & data.Swap.
+// Len implements sort.Sort(data Interface) for chainSlice - uses data.Len, data.Less & data.Swap.
```

Comment:
This reads more like documentation for `chainSlice` - move it up to that instead?

Reply:
Done.